app/components/post_with_channel_info/channel_info/channel_info.tsx

```diff
@@ -55,6 +63,7 @@ function ChannelInfo({channelName, teamName, testID}: Props) {
                 style={styles.channel}
                 testID='channel_display_name'
                 numberOfLines={1}
+                onPress={onChannelSwitch}
```

Comment:
Instead of directly the onPress, we may want to use a "Touchable" component, so it adds the expected touchable feedback. If you look for touchable in the codebase you will find many examples.

Reply:
Done. I have added touchable feedback with the type `opacity`.